test/zipObj.js

```diff
@@ -11,9 +11,8 @@ describe('zipObj', function() {
     eq(R.zipObj(['a', 'b', 'c'], [1, 2, 3, 4, 5, 6, 7]), {a: 1, b: 2, c: 3});
   });
 
-  it('extra keys are undefined', function() {
```

Comment:
Yes.  And I don't remember any particular discussion.  Obviously the function is useful, but I don't recall if there was anything in particular that drove us to add it.  I am guessing that the current behavior fell out of the implementation rather than from considered design, but I could be entirely wrong.